FSJavascript/Sign-UpForm/styles.css

```diff
@@ -0,0 +1,191 @@
+:root {
+  --white: rgb(238, 236, 236);
+  --light-gray: #f1f1f1;
+  --med-gray: #dcdddd;
+  --dark-gray: #3b3b3b;
+  --primary: #596d48;
+  --primary-emphasis: #283121;
+  --blue: blue;
+  --red: red;
+  font-family: sans-serif;
+}
+
+* {
+  box-sizing: border-box;
```

Comment:
Maybe consider moving the universal selector and box sizing border box to the reset.css or a global styles file